src/main/kotlin/org/rust/cargo/runconfig/RsExecutableRunner.kt

```diff
@@ -133,7 +135,7 @@ abstract class RsExecutableRunner(
         return DefaultExecutionResult(console, handler)
     }
 
-    open fun checkToolchainSupported(host: String): BuildResult.ToolchainError? = null
+    open fun checkToolchainSupported(toolchain: RsToolchain?, host: String): BuildResult.ToolchainError? = null
```

Comment:
Let's better pass `project` here to be consistent with `checkToolchainConfigured`